graphics/mesa/file_contexts

```diff
@@ -28,7 +28,9 @@
 /(vendor|system/vendor)/lib(64)?/dri/iris_dri\.so u:object_r:same_process_hal_file:s0
 /(vendor|system/vendor)/lib(64)?/dri/virtio_gpu_dri\.so u:object_r:same_process_hal_file:s0
 /(vendor|system/vendor)/lib(64)?/dri/gallium_dri\.so u:object_r:same_process_hal_file:s0
-/(vendor|system/vendor)/lib(64)?/hw/vulkan\.celadon\.so u:object_r:same_process_hal_file:s0
+/(vendor|system/vendor)/lib(64)?/hw/vulkan\.intel\.so u:object_r:same_process_hal_file:s0
+/(vendor|system/vendor)/lib(64)?/dri/libgallium_dri\.so u:object_r:same_process_hal_file:s0
+/(vendor|system/vendor)/lib(64)?/dri/libglapi\.so u:object_r:same_process_hal_file:s0
```

Comment:
Can you help confirm which folder is for libglapi now?
If it is located at /vendor/lib64/dri, then plz help remove line 26

Reply:
The glapi folder path is : /src/mapi/glapi, so will keep the change

Reply:
After double check, libglapi.so still in /vendor/lib64/, so will remove line 33. Thank you chenglei

Reply:
done.